diff --git a/gapless5.js b/gapless5.js
index 737a8b103712ca8a7ed0f38b818e0061e75a8291..26b688dd5f5eb0d915d65484e247616266029153 100644
--- a/gapless5.js
+++ b/gapless5.js
@@ -566,7 +566,7 @@ function Gapless5FileList(parentPlayer, parentLog, inShuffle, inLoadLimit = -1,
       return nextIndex;
     };
 
-    this.trackNumber = allowOverride ? updateShuffle(requestedIndex) : requestedIndex;
+    this.trackNumber = allowOverride ? requestedIndex : requestedIndex;
     log.debug(`Setting track number to ${this.trackNumber}`);
     this.updateLoading();
     player.scrub(0, true);
